admin.go

```diff
@@ -853,3 +871,23 @@ func (adm Admin) isClusterSetup(cluster string) (bool, error) {
 		keyBuilder.stateModelDefs(),
 	)
 }
+
+// createDataAccessorForPath checks that the cluster and path is valid to create a DataAcessor
+func (adm Admin) createDataAccessorForPath(
```

Comment:
I think it is fine to remove this func and do the check in an adhoc way. Other places in the repo follow the same way. If we want to refactor it, we should do it all. Maybe we can do it in another diff, if it is necessary.